explorer/app/components/Header/components/NavLinks/navLinks.tsx

```diff
@@ -0,0 +1,52 @@
+// Core dependencies
+import { Box, Button } from "@mui/material";
+import Link from "next/link";
+import React from "react";
+
+// App dependencies
+import { ELEMENT_ALIGNMENT } from "../../../../common/entities";
+
+export type NavAlignment = Omit<ELEMENT_ALIGNMENT, ELEMENT_ALIGNMENT.RIGHT>;
```

Comment:
My apologies, it looks like `Exclude` is the correct utility type to use here! You can use`Omit` is for interfaces and `Exclude` for union types and enums. See [Omit](https://www.typescriptlang.org/docs/handbook/utility-types.html#omittype-keys) and [Exclude](https://www.typescriptlang.org/docs/handbook/utility-types.html#excludeuniontype-excludedmembers) docs.